### ISSUE TITLE
explain how to build this one :)

### ISSUE THREAD

**Issue by 0xAwali** [MrRooten/GtScanner#1]
can you add section to explain how to build this one ?

**Comment by MrRooten** [comment]
Just add to idea or use gradle run the fatjar task.Later I will write a doc for it in README.md

**Comment by MrRooten** [comment]
And I have test it on open jdk17.I did not test it on any other jdk version,but I suppose it will build successfully above of jdk11 but may not run well in burpsuite

**Comment by 0xAwali** [comment]
yup I have  jdk17 and I tried to use gradle fatjar but not working !
